it/api/endpoints/lossClaim/list/v3/ListLossClaimsControllerISpec.scala

```diff
@@ -236,7 +270,76 @@ class ListLossClaimsControllerISpec extends V3IntegrationBaseSpec {
+      "querying for a specific typeOfLoss for a TYS tax year" in new TysIfsTest {
+        override val typeOfLoss: Option[String] = Some("uk-property-non-fhl")
+
+        val downstreamResponse: JsValue =
+          Json.parse(s"""[
+               |    {
+               |        "incomeSourceId": "XAIS12345678910",
+               |        "incomeSourceType": "02",
+               |        "reliefClaimed": "CSGI",
+               |        "taxYearClaimedFor": "2020",
+               |        "claimId": "AAZZ1234567890A",
+               |        "sequence": 1,
+               |        "submissionDate": "2020-07-13T12:13:763Z"
+               |    }
+               |]
+               |""".stripMargin)
+
+        override val responseJson: JsValue = Json.parse(s"""
+             |{
+             |    "claims": [
+             |        {
+             |            "businessId": "XAIS12345678910",
+             |            "typeOfLoss": "uk-property-non-fhl",
+             |            "typeOfClaim": "carry-sideways",
+             |            "taxYearClaimedFor": "2019-20",
+             |            "claimId": "AAZZ1234567890A",
+             |            "sequence": 1,
+             |            "lastModified": "2020-07-13T12:13:763Z",
+             |            "links" : [
+             |             {
+             |               "href" : "/individuals/losses/$nino/loss-claims/AAZZ1234567890A",
+             |               "rel": "self",
+             |               "method": "GET"
+             |             }
+             |            ]
+             |        }
+             |    ],
+             |    "links": [
+             |      {
+             |        "href": "/individuals/losses/$nino/loss-claims",
+             |        "rel": "self",
+             |        "method": "GET"
+             |      },
+             |      {
+             |        "href": "/individuals/losses/$nino/loss-claims",
+             |        "rel": "create-loss-claim",
+             |        "method": "POST"
+             |      },
+             |      {
+             |        "href": "/individuals/losses/$nino/loss-claims/order",
+             |        "rel": "amend-loss-claim-order",
+             |        "method": "PUT"
+             |      }
+             |    ]
+             |}""".stripMargin)
+
+        override def setupStubs(): StubMapping = {
+          AuditStub.audit()
+          AuthStub.authorised()
+          MtdIdLookupStub.ninoFound(nino)
+          DownstreamStub.onSuccess(DownstreamStub.GET, downstreamUrl, Map("incomeSourceType" -> "02"), Status.OK, downstreamResponse)
+        }
+
+        val response: WSResponse = await(request().get())
+        response.json shouldBe responseJson
+        response.status shouldBe Status.OK
+        response.header("Content-Type") shouldBe Some("application/json")
+      }
+
+      "querying for specific taxYear and businessId" in new NonTysTest {
```

Comment:
I don't think overriding the taxYear here does anything, as it is already defined in the NonTysTest trait